src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-ExtendingDesignsWithExistingComponents.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Developer Introduction to the Infusion Framework - Extending Designs with Existing Components
+layout: default
+category: Tutorials
+---
+
+Infusion's origins are in research about how to better implement accessible, inclusive systems. Many features are designed to support the easier transformation of data into other forms of representation, so it can be experienced by audiences with diverse needs.
+
+Our community often refers to this as *multimodal design* (in this context, a "modality" is "a particular mode in which something exists or is experienced or expressed"), and many characteristics of Infusion are about:
+
+* increasing the capacity of a system to have different representations of the same content, or to transform content into a form preferable to the end user
+* increasing the capacity of a system to have new forms of representation developed and added as needs arise for them
+
+We currently have a "Hello, World!" component that can say "hello!"" by printing to a web page and logging to the console. What if we wanted it to talk as well?
+
+We'll extend the code further using an existing Infusion [text to speech](/infusion/development/TextToSpeechAPI.html) component, which should work in a modern text-to-speech supporting browser:
+
+<div class="infusion-docs-note">You can check out the [Live Example](http://codepen.io/waharnum/pen/vgyBbe?editors=1111) of the code below on [CodePen](http://codepen.io/)</div>
+
+```
+fluid.defaults("fluid.helloWorld.sayHello", {
+    gradeNames: ["fluid.modelComponent"],
+    model: {
+        message: "Hello, world!"
+    },
+    modelListeners: {
+        "message": "{that}.sayHello"
+    },
+    invokers: {
+        sayHello: "fluid.notImplemented"
+    }
+});
+
+fluid.defaults("fluid.helloWorld.consoleHello", {
+    gradeNames: ["fluid.helloWorld.sayHello"],    
+    invokers: {
+        sayHello: {
+            "this": "console",
+            "method": "log",
+            "args": ["{that}.model.message"]
+        },
+    }
+});
+
+fluid.defaults("fluid.helloWorld.displayHello", {
+    gradeNames: ["fluid.helloWorld.sayHello", "fluid.viewComponent"],
```

Comment:
The introductory material on picking a grade doesn't mention inheritance.  You should probably mention that the order matters, with a simple example.  If it's still true that `fluid.viewComponent` needs to be the rightmost grade, that also needs to be mentioned.